src/Asset/EntrypointLookup.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+/**
+ * Returns the CSS or JavaScript files needed for a Webpack entry.
+ *
+ * This reads a JSON file with the format of Webpack Encore's entrypoints.json file.
+ * @final
+ */
+class EntrypointLookup
+{
+    private $entrypointJsonPath;
+
+    private $entriesData;
+
+    private $returnedFiles = [];
```

Comment:
this class needs to be Resettable, to reset this array. Otherwise it will cause issues when rendering multiple responses in the same PHP process (during tests, or when using a php app server rather than php-fpm)

Reply:
Good call. I've added it, though we can't rely on the new ResettableInterface from symfony/contracts yet, as it's not released

Reply:
well, as this is still a WIP, I think we could delay the stable release of this package until the release of symfony/contracts in November :smile: